sympy/solvers/ode/tests/test_single.py

```diff
@@ -1655,6 +1655,11 @@ def _get_examples_ode_sol_2nd_linear_bessel():
         'eq': (x-2)**2*(f(x).diff(x, 2)) - (x-2)*f(x).diff(x) + 4*(x-2)**2*f(x),
         'sol': [Eq(f(x), (x - 2)*(C1*besselj(1, 2*x - 4) + C2*bessely(1, 2*x - 4)))],
     },
+    #https://github.com/sympy/sympy/issues/4414
+    '2nd_lin_bessel_11': {
+        'eq': f(x).diff(x, x)+2/x*f(x).diff(x)+f(x),
+        'sol': [Eq(f(x), (C1*besselj('1/2', x) + C2*bessely('1/2', x))/sqrt(x))],
```

Comment:
There should be a blank line above the comment (and a space between `#` and `h`).

The equation would be easier to read with spaces before and after `+`.

We should not depend on string input to functions so use `S(1)/2` like the tests above.

Reply:
Sorry for the trouble, I made the changes as specified.
Thanks